File: server/AppSupport.Data/Entities/Plane.cs
using System;
using System.Collections.Generic;

namespace AppSupport.Data.Entities
{
    public class Plane
    {
        public int Id { get; set; }
        public int OrganizationId { get; set; }
        public int Capacity { get; set; }
        public string Name { get; set; }

        public Organization Organization { get; set; }

        public IEnumerable<ManifestPlane> PlaneManifests { get; set; }
        public IEnumerable<TemplatePlane> PlaneTemplates { get; set; }
    }
}

File: client/core/api/models/plane.ts
import {
  FormBuilder,
  FormGroup,
  Validators
} from '@angular/forms';

import { ManifestPlane } from './manifest-plane';
import { Organization } from './organization';
import { TemplatePlane } from './template-plane';

export interface Plane {
  id: number;
  organizationId: number;
  capacity: number;
  name: string;

  organization: Organization;

  planeManifests: ManifestPlane[];
  planeTemplates: TemplatePlane[];
}

export const PlaneForm = (plane: Plane, fb: FormBuilder): FormGroup =>
  fb.group({
    id: [plane.id],
    organizationId: [plane.organizationId, Validators.required],
    capacity: [plane.capacity, Validators.required],
    name: [plane.name, Validators.required]
  });
